src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -119,6 +130,32 @@
                 </a>
               </td>
             {% endif %}
+
+            <td class="text-right partial-refund{% if product.isRefundable() == false %} hidden{% endif %}">
+              <div class="form-group">
+                <div class="col-lg-4">
+                  <div class="input-group">
+                    <strong>{{ form_label(partialRefundForm['quantity_' ~ product.orderDetailId]) }}</strong>
+                    {{ form_widget(partialRefundForm['quantity_' ~ product.orderDetailId]) }}
+                    <div class="input-group-append"><div class="input-group-text">/ {{ product.quantityRefundable }}</div></div>
+                  </div>
+                </div>
+                <div class="col-lg-8">
+                  <div class="input-group">
+                    <strong>{{ form_label(partialRefundForm['amount_' ~ product.orderDetailId]) }}</strong>
+                    {{ form_widget(partialRefundForm['amount_' ~ product.orderDetailId]) }}
+                    <div class="input-group-append"><div class="input-group-text">€</div></div>
```

Comment:
just spotted this hardcoded currency symbol, I'll fix that (and the other one in the same file)